src/sdk/map_client.js

```diff
@@ -578,7 +577,7 @@ class MapClient {
             const current_attached_pools = tier_before_move.mirrors.map(mirror => mirror.spread_pools.map(pool => String(pool._id))).flat();
 
             if (
-                !js_utils.compare_unordered(target_attached_pools, current_attached_pools, true) ||
+                !_.xor(target_attached_pools, current_attached_pools) ||
```

Comment:
_.xor returns an array, which is always evaluated as a true value, but we need to check if it is non empty.
```suggestion
                _.xor(target_attached_pools, current_attached_pools).length ||
```